package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "specification",
+  "version": "1.0.0",
+  "description": "FiatConnect API Specification",
+  "main": "index.js",
```

Comment:
what happens if you remove this?

Reply:
Name and version are required in a package.json - [Creating a package.json file](https://docs.npmjs.com/creating-a-package-json-file#required-name-and-version-fields); I removed some of the other fields in https://github.com/fiatconnect/specification/pull/41/commits/8da2611c5af1537847bcc5f79b69e87c39f82697.